draft-bbs-signatures.md

```diff
@@ -404,36 +417,50 @@ Procedure:
 
 1. (W, H0, H) = octets_to_point(PK)
 
-2. h = XOF(SK  || msg[i] || ... || msg[L])
+2. generators =  (H_s || H_d || H_1 || ... || H_L)
+
+3. domain = OS2IP(HASH(PK || L || generators || Ciphersuite_ID || header)) mod q
+
+4. if domain is 0, abort
 
-3. for rand_el in (e, s) do
+5. h = XOF(SK  || domain || msg[i] || ... || msg[L])
```

Comment:
I like doing element access via the square bracket syntax as you have done here, but is this not in consistent with line 434? I'd prefer we do it this way but it needs to be the same throughout the doc

Reply:
```suggestion
5. h = XOF(SK  || domain || msg_1 || ... || msg_L)
```